### PR TITLE
Update contribution guidelines regarding requests for musicians, sport clubs, universities, government agencies, ...

### LINKED ISSUE #4274: Dealing with requests for individuals, musicians, bands, sport clubs, universities, government agencies etc.
Most of the icons at simple-icons concern either software projects or big, international companies. But there are a number of other "brands" out there, that people are requesting to be added to simple-icons.

The purpose of this issue (based on the discussion in #4270) is to create a unified place to discuss what should be added to simple-icons and what not. And how to measure the popularity, as an Alexa rank or stars on GitHub might not be the best metric to measure popularity for every brand.

### Type of brands

Here is a list of types of "brands" that have been requested or might be requested:

Type | Related issues/PRs | Examples
--- | --- | ---
Musicians or bands | #4270 | Ed Sheeran, Kanye West, Rammstein
YouTubers, YouTube Channels | #2944 | [PewDiePie](https://www.youtube.com/user/PewDiePie/featured), [5-Minute Crafts](https://www.youtube.com/c/5MinuteCraftsYouTube/featured), ...
Sport clubs or sport organizations | #2406 #2791 (discussed in #2513) | FC Bayern München, Bundesliga, NBA, WWE, Olympic, ...
Universities or educational institutes |  - | MIT, Oxford, Stanford, ...
Governmental organizations / departments / programs / agencies | #1635 #2318 |  Gov.uk and FedRAMP (which are actually already part of simple-icons), US AirForce, NASA, ESA (European Space Agency), ...

What are your opinions on these ones? Which ones would you consider a brand? Which could be added to simple-icons?

### Popularity and how to measure it

- For musicians, bands and YouTubers/channels probably an Alexa rank won't be the best option to measure popularity.
I can image that for example some individuals are very popular on YouTube with millions of views and subscribers, but have a website that isn't visited that much (and therefore has a low Alexa rank). What do you think about another metric for musicians, band and YouTube channels? Could the number of subscribers or views be an adequate replacement for an Alexa rank in this case?
With GitHub stars (5k at least) we already have an alternative measurement to measure popular software projects with not much traffic to their websites.
Of course this would only apply to musicians/YouTubers that are using a logo. If they are just popular but don't have any logo, we can't add them anyway.

- I guess sport clubs, sport organizations and universities could be covered by the Alexa rank. They should have enough traffic to their sites, if they are popular enough. So we could add MIT and Oxford, but not that small university in rural India for example. Same with sport clubs/organizations: Olympic, WWE, FC Bayern München could be added by their Alexa rank, but not the small soccer team from the neighbour village.

- I guess we could measure governmental departments / agencies as well by their Alexa rank. But here I have some doubts concerning legal stuff. I'm not too sure if it's legal to publish for example the logo of the US Air Force on simple-icons. It shouldn't look like we are acting on their behalf.
Should we check this for every request individually or not allow any governmental stuff at all? That would mean to probably as well remove the [gov.uk icon](http://simpleicons.org/?q=gov.uk).

Opinions from @simple-icons/maintainers as well as from other contributors are highly welcome! 😊

### PR DESCRIPTION
**Pull request by fbernhart**
Closes #4274 

- Updates the contribution guidelines regarding the discussion in #4274.

- I've as well added a quick note about our other "measurement of popularity" - Stars on GitHub.
   > For GitHub projects, the amount of "stars" should be above 5k.

- Could someone from the @simple-icons/maintainers, whose native language is English (is there anyone? 😅 ), have a look at it?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
 We welcome icon requests. Before you submit a new issue please make sure the icon:
 
 * Has not already been requested. If you find an existing issue or pull request for the brand you're looking for then please add a reaction or comment to show your support.
-* Is of a _popular_ brand. For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k. For anything else, popularity will be judged on a case-by-case basis.
-* Isn't related to anything that provides an illegal service (e.g. piracy, malware, threatening material, spam, etc.).
+* Is of a _popular_ brand. For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k. For GitHub projects, the amount of "stars" should be above 5k. For anything else, popularity will be judged on a case-by-case basis.
```
Comment (ericcornelissen) [inline CONTRIBUTING.md:23]: Maybe convert these into a sublist as well?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
...
+* Doesn't fall into one of the following categories:
+    - Anything that provides an illegal service (e.g. piracy, malware, threatening material, spam etc.)
+    - Governmental agencies, programs, departments (e.g. US Air Force, FBI) BUT we do allow international organizations and NGOs with supranational interests (e.g. WHO, Unicef, NASA, ESA, ...)
```
Comment (ericcornelissen) [inline CONTRIBUTING.md:26]: Maybe we can make this a bit shorter, how about:

```suggestion
    - Governmental agencies, programs, departments. International organizations and NGOs with supranational interests (e.g. WHO and Unicef) are allowed.
```

Also removed NASA and ESA from this sentence as they're not international nor NGOs. Not sure how to mention them though? Perhaps a sublist one level deeper?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
...
+    - Flags
```
Comment (ericcornelissen) [inline CONTRIBUTING.md:27]: Maybe expand on this a bit, how about:

```suggestion
    - Symbols, including as flags and banners
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
...
+    - Sport Clubs (e.g. Bayern Munich, FC Barcelona, ...), BUT we do allow sport organizations (e.g. FIFA, NBA, Olympic etc.)  
+    - Anything related to "yearly releases" (e.g. FIFA World Cup 2022, NBA 2020, ...)
+    - Universities
```
Comment (ericcornelissen) [inline CONTRIBUTING.md:30]: Maybe a bit more broad, in before someone requests their high schools logo :laughing: 

```suggestion
    - Universities and other educational institutions
```

Comment (mondeja) [inline CONTRIBUTING.md:30]: Maybe adding here "educational institutions"?

```suggestion
    - Universities or educational institutions
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
...
+    - Musicians and bands
+    - YouTubers and YouTube Channels
```
Comment (ericcornelissen) [inline CONTRIBUTING.md:32]: I think we should make this a bit more broad, including any social media site. I'm not sure what a generic word for "channel" is...

```suggestion
    - Social media personalities and groups/channels
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -20,8 +20,17 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
...
+    - Any other brands representing individuals rather than an organization, company or product.
```
Comment (mondeja) [inline CONTRIBUTING.md:33]: If this item of the list ends in a dot (`.`), all should end equal (I think that it's better all with dot ending).

Comment (ericcornelissen) [inline CONTRIBUTING.md:33]: Maybe we can somehow capture "Musicians and bands" and "YouTubers and YouTube Channels"/"Social media personalities" in this one?

**Review by ericcornelissen — COMMENTED** [review]
Good first draft, thanks for including the GitHub threshold for popularity as well :slightly_smiling_face: English looks good to me, though I'm not native either (@PeterShaggyNoble is, but having mistakes is fine, it may even give someone an opportunity to contribute :+1:)

File: CONTRIBUTING.md (hunk 6 above)
Comment (fbernhart) [reply CONTRIBUTING.md]: I've removed the single dot now as well. Hope that's o.k. for you 😄

**Comment by fbernhart** [comment]
@ericcornelissen @mondeja  Feel free to have another look at it. I've tried to update it according to your suggestions.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -20,8 +20,23 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
...
-* Isn't related to anything that provides an illegal service (e.g. piracy, malware, threatening material, spam, etc.).
+* Is of a _popular_ brand:
+    - For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k.
+    - For GitHub projects, the amount of "stars" should be above 5k.
+    - For anything else, popularity will be judged on a case-by-case basis.
+* Doesn't fall into one of the following categories:
+    - Anything that provides an illegal service (e.g. piracy, malware, threatening material, spam etc.)
+    - Governmental agencies, programs, departments.
+       - Allowed: International organizations and NGOs with supranational interests (e.g. WHO and Unicef).
+       - Allowed: Organizations like NASA or ESA.
```
Comment (mondeja) [inline CONTRIBUTING.md:31]: These ends with dots... should be removed to maintain the consistency?

Comment (fbernhart) [reply CONTRIBUTING.md:31]: Sorry. 😅  I'll remove them.

**Review by runxel — APPROVED** [review]
Looking great!

**Review by ericcornelissen — COMMENTED** [review]
I'm not sure if I'm in favor of keeping all the examples :thinking: First off, a minor concern is that the examples match quite closely, though not exactly, the stereotype of a European male developer (EU soccer clubs and YouTube/Twitch rather than Instagram or the like)... Moreover, in most cases I don't even think the examples really clarify anything, as the text itself is clear. An exception to this is maybe for the "allowed" ones under governmental organisations. Though "Organisations like NASA or ESA" can be interpreted in many ways and should probably change to "Space agencies"

**Comment by fbernhart** [comment]
I wanted to give examples so that the categories become clearer, for example the difference between sport clubs and sport organizations.

@ericcornelissen if you want, we can add more "diverse" examples.

As I'm not into soccer at all, these clubs were just the ones that first came to my mind. 😄 Feel free to add any non-european soccer club of your choice.

**Comment by ericcornelissen** [comment]
My main point is definitely not that the examples are not diverse (making them diverse would be difficult anyway as you will always have to few examples to include everyone). More important is that I don't think the examples actually add any value. What constitutes a sport club is not made any clearer by mentioning FC Barcelona. If you want to use examples to differentiate Sports Clubs and Sports Organisations specifically I think it is more useful to explicitly compare them (e.g. FC Barcelona against FIFA or NY Yankees against NBL), though again I would say this distinction is clear enough without ever mentioning Sports Organisations - which you may disagree with.

**Comment by fbernhart** [comment]
@ericcornelissen I've removed the examples now. 😊 Feel free to have another final look at it.

**Review by mondeja — APPROVED** [review]
Great work :1st_place_medal:

**Review by service-paradis — APPROVED** [review]

**Review by ericcornelissen — APPROVED** [review]
Looks good to me now @fbernhart 👍 Given all the approvals I will merge this in.

> I've removed the examples now. 😊 Feel free to have another final look at it.

To be clear, if you and other maintainers think the examples are useful they can be added, I really just doubt they are but am open to discussion.

**Pull request merged**